contracts/compliance/modular/ModularCompliance.sol

```diff
@@ -114,6 +114,7 @@ contract ModularCompliance is IModularCompliance, OwnableUpgradeable, MCStorage
      */
     function addModule(address _module) external override onlyOwner {
         require(!moduleBound[_module], 'module already bound');
+        require(modules.length <= 25, 'cannot add more than 25 modules');
```

Comment:
```suggestion
        require(modules.length <= 24, 'cannot add more than 25 modules');
```

Otherwise it means we cas add 26 modules.

Reply:
```suggestion
        require(modules.length < 25, 'cannot add more than 25 modules');
```

Ot strict comparison.

Reply:
indeed